massa-network-worker/src/network_worker.rs

```diff
@@ -185,7 +184,7 @@ impl NetworkWorker {
                     ).await?
                 },
 
-                // incoming command
+                // incoming command from client
```

Comment:
```suggestion
                // incoming command
```

The commands come from both protocol and the client. But this was a choice to not explicit from where commands are coming as it could be from anywhere